packages/pd-manager-rest-api/src/api-functions.ts

```diff
@@ -0,0 +1,94 @@
+import { checkAuth, sendErrorResponse, ISingleEndpointOpts } from '@sphereon/ssi-express-support'
+import { Request, Response, Router } from 'express'
+import { IRequiredContext } from './types'
+import { AddDefinitionItemArgs, PersistDefinitionArgs, UpdateDefinitionItemArgs } from '@sphereon/ssi-sdk.pd-manager'
+import { DeleteDefinitionArgs } from '@sphereon/ssi-sdk.data-store'
+
+export function pdReadEndpoint(router: Router, context: IRequiredContext, opts?: ISingleEndpointOpts) {
+  if (opts?.enabled === false) {
+    console.log(`"pdReadEndpoint" Endpoint is disabled`)
+    return
+  }
+  const path = opts?.path ?? '/presentation-defs'
```

Comment:
/presentation-definitions?

Reply:
also maybe make a const for this? so you do not have to copy it everywhere and make mistakes?

Reply:
fixed